### DISCUSSION TITLE
Is it possible to slice objects with just shell commands or via Web Assembly?

### DISCUSSION THREAD

**Discussion by Cugatay** [OrcaSlicer/OrcaSlicer#3791] [Q&A]
Hello, I'd like to develop some software for fun that requires 3d model slicing.
Is it possible to slice 3d models with just a shell command?
Or is there any Web Assembly version of Orca Slicer? It would be even better if any wasm version of the slicer existed.

**Comment by ToyboxZach** [comment]
I was able to achieve it with Prusa https://github.com/ToyboxZach/PrusaSlicer
But Orca's code is a bit more complicated to separate out the UI, so my attempts to build orca slicer failed.
